x/oracle/keeper/grpc_query.go

```diff
@@ -0,0 +1,214 @@
+package keeper
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// querier is used as Keeper will have duplicate methods if used directly, and gRPC names take precedence over q
+type querier struct {
+	Keeper
+}
+
+// NewQuerier returns an implementation of the oracle QueryServer interface
+// for the provided Keeper.
+func NewQuerier(keeper Keeper) types.QueryServer {
+	return &querier{Keeper: keeper}
+}
+
+var _ types.QueryServer = querier{}
+
+// Params queries params of x/oracle module
+func (q querier) Params(c context.Context, req *types.QueryParamsRequest) (*types.QueryParamsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	var params types.Params
+	q.paramSpace.GetParamSet(ctx, &params)
```

Comment:
There should be a method on the keeper to get the Params. e.g. https://github.com/umee-network/umee/blob/main/x/leverage/keeper/grpc_query.go#L51